src/pages/app.tsx

```diff
@@ -61,18 +71,22 @@ const App = () => {
             variant: 'layout.main',
           }}
         >
-          <Container
-          // sx={{
-          //   mt: 3,
-          // }}
-          // maxWidth="sm"
-          >
-            <Router basepath="/app">
-              <TracksContainer path="/" />
-              <TrackContainer path=":trackId" />
-              <Profile path="/profile" me={user.authUser} signOut={user.signOut} />
+          <Container sx={{ mt: '64px' }} maxWidth="xl">
+            <Router basepath="app">
+              <GenericRouteWrapper path="tracks">
+                <Fragment>
```

Comment:
What do you think about puting the placement inside GenericRouteWrapper?

Reply:
What is the placement?

Reply:
`Fragment`, sorry